lib/views/pages/partners.html

```diff
@@ -73,6 +73,6 @@
                 </a>
             </li>
         </ul>
+        -->
```

Comment:
As it was commented out before, I just adapted. It´s likely that well have some more partners for the raffle on board soon, so I´ll say, lets just keep it.